java/xen-api-samples/src/main/java/com/xensource/xenapi/samples/GetVariousRecords.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) Citrix Systems, Inc.
```

Comment:
Citrix or Cloud Software Group Holdings, Inc. ?

Not sure when this lingo switch is supposed to occur.

Reply:
Cloud - it can count as a new file. I'll correct it.